rome-modules/src/test/java/com/rometools/modules/mediarss/MediaModuleTest.java

```diff
@@ -91,6 +89,30 @@ public void testParse() throws Exception {
         }
     }
 
+
+    public void testParseFileSizeWithUnits() throws Exception {
+        final SyndFeed feed = getSyndFeed("org/rometools/feed/module/mediarss/filesize-with-unit.xml");
```

Comment:
Maybe make `parseFileSize` static and package-private? Then you could test it thoroughly here without any xml files. When the method is tested, you could add one test for the integration, i.e. with xml file and `SyndFeed`.